open-banking-accelerator/components/consent-management/com.wso2.openbanking.accelerator.consent.extensions/src/main/java/com/wso2/openbanking/accelerator/consent/extensions/common/ConsentExtensionConstants.java

```diff
@@ -153,19 +153,38 @@ public class ConsentExtensionConstants {
     public static final String ACCOUNTS_SELF_LINK = "Consent.AccountAPIURL";
     public static final String PAYMENT_SELF_LINK = "Consent.PaymentAPIURL";
     public static final String COF_SELF_LINK = "Consent.FundsConfirmationAPIURL";
+    public static final String VRP_SELF_LINK = "Consent.VRPAPIURL";
     public static final String REVOKED_STATUS = "revoked";
-
     public static final String DISPLAY_NAME = "display_name";
     public static final String ACCOUNT_DATA = "account_data";
     public static final String SELECTED_ACCOUNT = "selectedAccount";
     public static final String PAYMENT_COF_PATH = "funds-confirmation";
-
     public static final String AWAITING_UPLOAD_STATUS = "awaitingUpload";
-
     public static final String OB_REVOKED_STATUS = "Revoked";
     public static final String OB_REJECTED_STATUS = "Rejected";
     public static final String OB_AUTHORIZED_STATUS = "Authorised";
     public static final String OB_AWAITING_AUTH_STATUS = "AwaitingAuthorisation";
     public static final String OB_AWAITING_UPLOAD_STATUS = "AwaitingUpload";
-
+    public static final String VRP_CONSENT_PATH = "domestic-vrp-consents";
```

Comment:
Shall we separate these params for more readability?

Reply:
Fixed in [3c0b5a6]